code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -5,14 +5,24 @@
 
 /obj/structure/closet/crate/necropolis
 	name = "necropolis chest"
-	desc = "It's watching you closely."
+	desc = "It's watching you closely. You need a skeleton key to open it."
 	icon_state = "necrocrate"
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
 /obj/structure/closet/crate/necropolis/tendril
 	desc = "It's watching you suspiciously."
+	integrity_failure = 0 ///prevents bust_open to fire
+	var/spawned_loot = FALSE /// var to check if it got opened by a key
 
-/obj/structure/closet/crate/necropolis/tendril/PopulateContents()
+/obj/structure/closet/crate/necropolis/tendril/Initialize()
+	. = ..()
+	RegisterSignal(src, COMSIG_PARENT_ATTACKBY, .proc/try_spawn_loot)
```

Comment:
Why a signal instead of just overriding `/attackby`?

Reply:
because when i overwrite attackby maints say to use signalers

Reply:
uhhh ok

Reply:
Follow-up question: If `/obj/structure/closet/crate/necropolis` has "You need a skeleton key to open it." in its description, why is this registered on the subtype of that?